sched/misc/assert.c

```diff
@@ -209,11 +209,78 @@ static void showstacks(void)
 
 #endif
 
+/****************************************************************************
+ * Name: get_argv_str
+ *
+ * Description:
+ *   Safely read the contents of a task's argument vector, into a a safe
+ *   buffer.
+ *
+ ****************************************************************************/
+
+static void get_argv_str(FAR struct tcb_s *tcb, FAR char *args, size_t size)
+{
+  /* Perform sanity checks */
+
+  if (!tcb || !tcb->group || !tcb->group->tg_info)
+    {
+      /* Something is very wrong -> get out */
+
+      *args = '\0';
+      return;
+    }
+
+#ifdef CONFIG_ARCH_ADDRENV
+  save_addrenv_t oldenv;
+  bool saved = false;
+
+  if ((tcb->flags & TCB_FLAG_TTYPE_MASK) != TCB_FLAG_TTYPE_KERNEL)
+    {
+      if ((tcb->group->tg_flags & GROUP_FLAG_ADDRENV) == 0)
+        {
+          /* Process should have address environment, but doesn't */
+
+          *args = '\0';
+          return;
+        }
+
+      up_addrenv_select(&tcb->group->tg_addrenv, &oldenv);
+      saved = true;
+    }
+#endif
+
+#ifndef CONFIG_DISABLE_PTHREAD
+  if ((tcb->flags & TCB_FLAG_TTYPE_MASK) == TCB_FLAG_TTYPE_PTHREAD)
+    {
+      struct pthread_tcb_s *ptcb = (struct pthread_tcb_s *)tcb;
```

Comment:
```suggestion
      FAR struct pthread_tcb_s *ptcb = (FAR struct pthread_tcb_s *)tcb;
```

Reply:
Thanks, I fixed a couple of missing FAR qualifiers but missed these (this code is copy&paste).